core/src/main/java/com/google/googlejavaformat/InputOutput.java

```diff
@@ -82,7 +82,7 @@ protected final void computeRanges(List<? extends Input.Tok> toks) {
         for (int i = lineI0; i <= lineI; i++) {
           addToRanges(ranges, i, k);
         }
-        addToRanges(range1s, lineI0, k);
+        addToRanges(range1s, lineI, k);
```

Comment:
@winteryoung Could you explain for me why you changed the argument from `lineIO` to `lineI` here? It's not immediately obvious to me why you think it's correct.

Reply:
The original code is
```java
      if (k >= 0) {
        addToRanges(range0s, lineI0, k);
        for (int i = lineI0; i <= lineI; i++) {
          addToRanges(ranges, i, k);
        }
        addToRanges(range1s, lineI0, k);
      }
```
The first `addToRanges()` and the last `addToRanges()` make no difference. The original one could pass the tests. I guess that's because there's no test cases contain tokens that span multiple lines.

Reply:
This doesn't look right to me. I think this doesn't break tests because `range0s` isn't being used, and `range1s` is being used in a situation where `ranges` would also be correct.

Reply:
Then what's the difference between `range0s` and `range1s`? They contain the same contents.

Reply:
Ah, thanks. This does look like a mistake, but the reason it wasn't breaking tests is that it wasn't really being used. I removed it in c48ee99f0240e0e51af842b3cf17793a83508bd4.